app/controllers/posts_controller.rb

```diff
@@ -8,4 +8,33 @@ def show
     @post = Post.where(author_id: params[:user_id], id: params[:id]).first
     @comments = Comment.where(post: @post)
   end
+
+  def new
+    post = Post.new
+    respond_to do |r|
+      r.html { render :new, locals: { post:, author: current_user } }
+    end
+  end
+
+  def create
+    post = Post.new(post_params)
```

Comment:
- [ ] As pointed out by the previous reviewer, please ensure that every new post is created on behalf of the new user(not during rendering stage)

Reply:
![image](https://user-images.githubusercontent.com/44860643/195934883-275b15aa-e330-4e9a-a708-76f6ce501907.png)
fixed